src/Microsoft.Health.SqlServer/Routing/KnownRoutes.cs

```diff
@@ -15,10 +16,17 @@ public static class KnownRoutes
     public const string Versions = "versions";
 
     public const string Current = Versions + "/current";
-    public const string Script = Versions + "/{id:int}/script";
+    public const string IdSegment = "{id:int}";
+    public const string Script = Versions + "/" + IdSegment + "/script";
     public const string Diff = Script + "/diff";
 
     internal static readonly Uri RootedCurrentUri = new Uri("/" + SchemaRoot + "/" + Current, UriKind.Relative);
     internal static readonly Uri RootedCompatibilityUri = new Uri("/" + SchemaRoot + "/" + Compatibility, UriKind.Relative);
     internal static readonly Uri RootedVersionsUri = new Uri("/" + SchemaRoot + "/" + Versions, UriKind.Relative);
+
+    internal static Uri RootedScriptUri(int version) => 
```

Comment:
nit: I think our style has been to put `=>` on the next line for methods

Reply:
Fixed in latest commit.